x-pack/plugins/observability/public/components/shared/exploratory_view/configurations/synthetics/single_metric_config.ts

```diff
@@ -93,6 +93,30 @@ export function getSyntheticsSingleMetricConfig({ dataView }: ConfigProps): Seri
           titlePosition: 'bottom',
         },
       },
+      {
+        id: 'monitor_total_runs',
+        label: i18n.translate('xpack.observability.expView.totalRuns', {
+          defaultMessage: 'Total Runs',
+        }),
+        metricStateOptions: {
+          titlePosition: 'bottom',
+        },
+        columnType: FORMULA_COLUMN,
+        formula: 'unique_count(state.id)',
```

Comment:
```suggestion
        formula: 'unique_count(monitor.check_group)',
```

Reply:
i am sorry for mis-guiding you on this. Sate.id actually remains same, in case the monitor state doesn't change. 

monitor.check_group is unique for every new test run.

Reply:
Shall I live it like it is then?

Reply:
no, we do need monitor.check_group instead of state.id